### PR TITLE
Cross browser compatibility issues

### PR DESCRIPTION
**Pull request by KRRISH96**
Fix cross browser issues.

### CODE REVIEW

**Comment by KRRISH96** [comment]
@prasanna1211 
Please Do not merge this PR until all other `fix`, `1.x target` and `design-update` PR's are merged.

Merging could cause conflicts in other PRs (So, will rebase this PR once all other PRs are merged ).

**Comment by KRRISH96** [comment]
Rebased on latest development.
Can be merged now.

**Review by rishichawda — APPROVED** [review]

**Pull request merged**
